tests/tests.yaml

```diff
@@ -0,0 +1,167 @@
+&root
+- name: Self-test
+  description: Test the schema that validates this file
+  valid: true
+  schema: tests
+  version: 0
+  data: *root
```

Comment:
Why not adding the testcases as typescript objects at the top of the index.ts file ? Adding a fullblown schemas to just validate testcase feels a bit overengineered.

Reply:
Have to admit I did not think of that, though it does seem like the logical solution. The idea of a self-test here sounded cool in my mind, so I went ahead and did it.

Reply:
Done in 21980a9ab337356a371d16871242c37e0645f832